tests/unit/js/init.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-useless-constructor */
 /**
  * init.js : Configuration for the library require.js
```

Comment:
This old description is out of date as we don't use require.js any more. Please change the description to "Global configuration of the app".